tests/test_render.py

```diff
@@ -154,11 +154,12 @@ def test_initial_state(self):
         self.startServer(app)
         el = self.wait_for_element_by_css_selector('#react-entry-point')
 
+        # Note: this .html file shows there's no undo/redo button by default
```

Comment:
Due to visual tests instability I would prefer to isolate this behavior in its own test instead of grafting it to an existing one. Specifically, testing that undo & redo buttons are not found in the DOM + a percy snapshot.

Reply:
I'll include a test that these elements aren't in the DOM. The test I already added includes a percy snapshot with undo AND redo buttons - I guess for completeness I could extend it test that at either end of the history the appropriate button disappears, I'll do that - but we now have a million percy snapshots that used to have undo buttons and no longer do.

Reply:
extended element existence tests -> https://github.com/plotly/dash-renderer/pull/175/commits/56cfddc62807208eadb4ad18378bd0a37508081c